cosmic-applet-workspaces/src/components/app.rs

```diff
@@ -114,6 +117,12 @@ impl cosmic::Application for IcedWorkspacesApplet {
                     let _ = tx.try_send(WorkspaceEvent::Scroll(delta));
                 }
             }
+            Message::WorkspaceOverview => {
+                let _ = ShellCommand::new("sh")
+                    .arg("-c")
+                    .arg("cosmic-workspaces")
+                    .spawn();
```

Comment:
Is `sh` needed when there are no shell language arguments?

Reply:
Good point! Fixed